client/src/components/DateTime.vue

```diff
@@ -1,8 +1,18 @@
 <script lang="ts" setup>
 import moment from "moment";
-import "moment/min/locales";
+import "moment/dist/locale/es";
+import "moment/dist/locale/ar";
+import "moment/dist/locale/fi";
+import "moment/dist/locale/da";
+import "moment/dist/locale/de";
+import "moment/dist/locale/fr";
+import "moment/dist/locale/sv";
+import "moment/dist/locale/is";
+import "moment/dist/locale/ru";
+import "moment/dist/locale/nl";
```

Comment:
It is no longer possible to get all the locales on the same import so I'm picking the ones we offer support on Coference.